### ISSUE TITLE
[catalog] No description of `Language` field

### ISSUE THREAD

**Issue by funnelfiasco** [ossf/sbom-everywhere#63]
The SBOM Catalog's [contributing docs](https://github.com/ossf/sbom-everywhere/tree/main/SBOM-Catalog/contribute.md) don't mention the *Language* field that appears in `data.yaml`. From existing entries, it seems to be "what language ecosystems does this tool work in?" but it's not entirely clear. 

In #62, I used "generic" because I saw other examples of that, but I'm not sure if that's...right.

**Comment by funnelfiasco** [comment]
The tool tip on the catalog itself says:

> Filter to distinguish between different programming languages, build tools or operating system

But it's not clear if the language a tool is written in applies here. Arguably, these should be three separate values (e.g. *Language*, *SupportedTool*, and *SupportedOS*)

**Comment by joshbressers** [comment]
Would I be correct in assuming #65 resolves this?

**Comment by funnelfiasco** [comment]
No, #65 is for the License field, but this one is for Language.
